Jagati.cmake

```diff
@@ -69,7 +69,7 @@ if(JagatiVersion)
     message(STATUS "Already loaded Jagati version '${JagatiVersion}', not loading again.")
```

Comment:
I am assuming this message is only displayed if the JagatiVersion does not need to be updated.

Reply:
Since the Jagati implies many different packages with similar starts and ends (because they are part of the same interconnected system) and we don't know which package is going to be first in the line, we do a sanity check in each one to prevent issues with different Jagati versions being loaded.